src/healthcheck.js

```diff
@@ -15,9 +15,13 @@ function fetchBestBlock(db): Promise<any> {
 
 async function start(db: any) {
   logger.debug('start')
+
+  process.env.DATABASE_UNHEALTHY = 'false'
```

Comment:
I would remove the DATABASE_UNHEALTHY variable and keep only DATABASE_UNHEALTHY_SINCE. Also I would not specify the UNHEALTHY_LIMIT this way, it's used once anyway so I would hardcode it's value where it is used for the sake of readability. There's no need to tinker much with that value once it's properly set.